SPECS-CROSS/cross-gcc/cross-gcc.spec

```diff
@@ -0,0 +1,161 @@
+%global debug_package %{nil}
```

Comment:
Why did we need to drop the debug package? We can skip stripping the symbols, but we'll still be missing the sources, which make debugging a lot easier.